tests/unit_tests/test_rating_service.py

```diff
@@ -343,3 +343,20 @@ async def test_game_rating_error_handled(rating_service, game_info, bad_game_inf
     service._logger.warning.assert_called()
     # second game: results have been saved.
     service._persist_rating_changes.assert_called_once()
+
+
+async def test_game_update_empty_resultset_raises_exception(semiinitialized_service):
```

Comment:
I don't think you want semi-initialized service here. I think you want the normal fully initialized service.